src/kubeChannel.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from "vscode";
+
+export const kubeChannel = {
+    channel: vscode.window.createOutputChannel("Kubernetes"),
+
+    showOutput(message: any, title?: string) {

```

Comment:
I've been preferring to export interfaces for this purpose, so that they can be mocked/faked in tests.  At the moment this is only used in `extension.ts`, though, so we can defer doing that.

Reply:
nice.